backend/manager/views.py

```diff
@@ -58,6 +59,12 @@ class CrawlerList(generics.ListCreateAPIView):
     permission_classes = (IsAdminUser, )
 
     def perform_create(self, serializer):
+        crawlers = Crawler.objects.all()
+        for c in crawlers:
+            status = c.status.split()[0]
+            if status == 'creating' or status == 'running:':
```

Comment:
You can use `filter` method for searching instances with some params, like this,
```python3
if Crawler.objects.filter(status__contains='creating').exists() or Crawler.objects.filter(status__contains='running').exists():
```